### ISSUE TITLE
Unable to compile TypeScript: teardown.js' is not under 'rootDir'

### ISSUE THREAD

**Issue by bnwan** [wallabyjs/quokka#44]
​​Unable to compile TypeScript​​
​​File 'C:/Users/BenN/.vscode/extensions/WallabyJs.quokka-vscode-1.0.30/dist/wallaby/runners/node/teardown.js' is not under 'rootDir' 'c:/Projects/FrontEnd/src'. 'rootDir' is expected to contain all source files. (6059)​​

### Code editor version

Visual Studio Code v1.11.2

### OS name and version

Windows

**Comment by ArtemGovorov** [comment]
Can't reproduce the issue on a simple TS file. 
Please share
- your quokka file (that you're trying to run),
- your package.json file,
- your tsconfig.json file.

**Comment by breathe** [comment]
I see the same issue -- it happened after 

(1) starting up quokka in vscode in a file
(2) doing other things in the project for awhile
(3) then after awhile I saw the error message printed in the vscode output

scratchpad.quoakka.ts
```
/* tslint:disable */
import { nonenumerable, excludeNil } from "./src/Helpers/Utilities";

class Foo {}

new Foo().constructor.name; /*?*/

12; /*?*/
```

tsconfig.json
```
{
  "compileOnSave": true,
  "compilerOptions": {
    "rootDir": "./src",
    "outDir": "./lib",
    "lib": ["es2017"],
    "strict": true,
    "strictNullChecks": true,
    "noImplicitAny": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "module": "commonjs",
    "target": "es2015",
    "allowJs": true,
    "experimentalDecorators": true,
    "inlineSourceMap": true,
    "inlineSources": true,
    "forceConsistentCasingInFileNames": true
  },
  "include": ["./src"]
}
```

excerpt from package.json
```
{
  "compileOnSave": true,
  "compilerOptions": {
    "rootDir": "./src",
    "outDir": "./lib",
    "lib": ["es2017"],
    "strict": true,
    "strictNullChecks": true,
    "noImplicitAny": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "module": "commonjs",
    "target": "es2015",
    "allowJs": true,
    "experimentalDecorators": true,
    "inlineSourceMap": true,
    "inlineSources": true,
    "forceConsistentCasingInFileNames": true
  },
  "include": ["./src"]
}
```

**Comment by ArtemGovorov** [comment]
@breathe Thanks for providing the details!
The issue is fixed and the fix is published in v1.0.47.
